class GettingUsersError(Exception):
    pass

class KeyIsNotInDataError(Exception):
    pass
